crates/data_components/src/cdc.rs

```diff
@@ -62,21 +64,27 @@ pub trait CommitChange {
 }
 
 pub struct ChangeEnvelope {
-    change_committer: Box<dyn CommitChange + Send>,
+    change_committer: Option<Box<dyn CommitChange + Send>>,
```

Comment:
I would prefer we leave this as not an option, and just define a no-op one for the Spice stream